tce/src/applibs/LLVMBackend/LLVMBackend.cc

```diff
@@ -884,10 +884,9 @@ LLVMBackend::createPlugin(const TTAMachine::Machine& target) {
     }
 
     // NOTE: this could be get from Makefile.am
-    TCEString pluginSources = 
-	srcsPath + "PluginCompileWrapper.cc " +
-	srcsPath + "TCEFrameInfo.cc " +
-	srcsPath + "TCETargetMachinePlugin.cc ";
+    TCEString pluginSources = srcsPath + "PluginCompileWrapper.cc " +
```

Comment:
Sorry, but this is still not matching the old TCE style which would be
```
TCEString pluginSources = 
   srcsPath + "TCEFrameInfo.cc " + srcsPath + ... (until 78)
   srcPath + ... (until 78)  etc.
```

Reply:
Can you check if the clang-format supports that somehow?

Reply:
Hey Pekka. I looked into this. I read through all the clang-format options here: https://clang.llvm.org/docs/ClangFormatStyleOptions.html . But I didn't find an option that was doing this. I found one that was close to it "AlwaysBreakBeforeMultilineStrings", but not exactly what we want. In the end I asked a question on stackoverflow about it. https://stackoverflow.com/questions/59235609/clang-format-multiline-variable-assignments . Some comments were made but no answers. So that's as far as I want to go for this issue. I'll put an enter there manually. In the future I'll see if someone provides an answer on stackoverflow and I could apply a format script fix later.